postgres/src/main/kotlin/com/example/plugins/Routing.kt

```diff
@@ -0,0 +1,69 @@
+package com.example.plugins
+
+import com.example.exceptions.DbElementNotFoundException
+import com.example.models.Article
+import com.example.service.ArticleService
+import io.ktor.http.*
+import io.ktor.server.application.*
+import io.ktor.server.config.yaml.*
+import io.ktor.server.request.*
+import io.ktor.server.response.*
+import io.ktor.server.routing.*
+import java.sql.*
+
+fun Application.configureRouting(dbConnection: Connection) {
+    val articleService = ArticleService(dbConnection)
+    routing {
+        // Create new Article
+        post("/articles") {
+            val article = call.receive<Article>()
+            val id = articleService.create(article)
+            call.respond(HttpStatusCode.Created, id)
+        }
+        // Read an Article
+        get("/articles/{id}") {
+            val id = call.parameters["id"]?.toInt() ?: throw DbElementNotFoundException("Invalid article ID")
```

Comment:
If a requested article ID isn't a whole number, then the article isn't found in the database, right?

Reply:
yes, that's right